### PR TITLE
[ISSUE-382] Labels for the logging and crashdump folder

### PR DESCRIPTION
**Pull request by painhardcore**
### Issue #382 

Ephemeral folder for crashdump
And label for the logging system to detect which release deployed

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
_Link to custom CI build_

### CODE REVIEW

**Review by yurkov-anton — APPROVED** [review]

File: charts/csi-baremetal-driver/templates/controller.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ spec:
         app.kubernetes.io/name: csi-baremetal
         app: csi-baremetal-controller
         role: csi-do
+        release: {{ .Release.Name }}
```
Comment (mishoyama) [inline charts/csi-baremetal-driver/templates/controller.yaml:19]: Please put comments why this option is present in charts and make corresponding PR to operator.

**Review by mishoyama — CHANGES_REQUESTED** [review]

**Review by safronovD — APPROVED** [review]

File: charts/csi-baremetal-driver/templates/controller.yaml (hunk 1 above)
Comment (painhardcore) [reply charts/csi-baremetal-driver/templates/controller.yaml:19]: comment in every chart?

**Review by AnastasiaShemyakinskaya — APPROVED** [review]

**Comment by painhardcore** [comment]
https://github.com/dell/csi-baremetal-operator/pull/27 the same changes in the operator!

File: charts/csi-baremetal-driver/templates/controller.yaml (hunk 1 above)
Comment (mishoyama) [reply charts/csi-baremetal-driver/templates/controller.yaml:19]: These charts will be deprecated soon. Please add comment to the chart in Operator repo.

**Review by mishoyama — APPROVED** [review]

**Pull request merged**